src/components/ui/Breadcrumb.tsx

```
@@ -24,7 +25,13 @@ const BreadcrumbContainer = styled(FlexRow)`
 `;
 
 export default function Breadcrumb(props: BreadcrumbProps) {
-  return <BreadcrumbContainer children={props.children} style={props.style} />;
+  return (
+    <BreadcrumbContainer
+      class={props.class}
+      children={props.children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)